requirements.txt

```diff
@@ -1,3 +1,3 @@
 sphinx>2,<3
-sphinx-hoverxref
+git+git://github.com/readthedocs/sphinx-hoverxref@0.4b1#egg=sphinx-hoverxref
```

Comment:
Why use the GitHub version since `sphinx-hoverxref` has been updated to 0.4b1 on PyPI? IOW, the original requirement should be good enough.

Reply:
See your earlier point https://github.com/cellml/cellml-specification/pull/315#discussion_r416293885 which is why this PR has been waiting in the first place ... ?

Reply:
?? Back then, the official version of `sphinx-hoverxref` was `0.3b1`, which wasn't working for us, hence indeed my suggestion to use a specific revision of `sphinx-hoverxref` from their GitHub repo (rather than the version on the `master` branch, which might be broken or something while I would expect a release to be fine; as per my [comment](https://github.com/cellml/cellml-specification/pull/315#discussion_r416293885)). Either way, now, the official version of `sphinx-hoverxref` is `0.4b1`, which is what we want. So, I would prefer to use the official version (as per my [original](https://github.com/cellml/cellml-specification/pull/315#discussion_r416293885) and [second](https://github.com/cellml/cellml-specification/pull/315#discussion_r416301541) comments).

Reply:
Ah, ok, I get it.  I thought you wanted it pinned to a release.  I'll change it now.